package.json

```diff
@@ -14,12 +14,13 @@
     "@redhat-cloud-services/rbac-client": "^1.0.31",
     "awesome-debounce-promise": "^2.1.0",
     "axios": "^0.18.0",
+    "node-fetch": "^2.6.0",
```

Comment:
This library does not seem to used anywhere.

Reply:
There were errors on the travis test indicating this was missing, so I've added it - I will try again without it.

Reply:
Tried again without this library - while my local testing works, the travis build fails with these type of errors:
FAIL  src/test/presentional-components/shared/pf4-select-wrapper.test.js

  ● Test suite failed to run

    Cannot find module 'node-fetch' from 'server.js'

      at Resolver.resolveModule (node_modules/jest-resolve/build/index.js:229:17)

      at Object.<anonymous> (node_modules/fetch-mock/src/server.js:1:104)

Reply:
I added the node-fetch back.

Reply:
Oh i see. Does this app still use fetch/fetch-mock somewhere? That is the source of this issue. I will do some audit once i am back from PTO.